scripts/dev-logs.sh

```
@@ -0,0 +1,43 @@
+        SERVICE=$arg
+    fi
+done
+
+# Colors
+BLUE='\033[0;34m'
+YELLOW='\033[1;33m'
+NC='\033[0m'
+
+if [ -z "$SERVICE" ]; then
+    echo -e "${BLUE}Viewing logs for all services (Ctrl+C to exit)${NC}"
+    docker compose logs -f $EXTRA_ARGS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)